### DISCUSSION TITLE
CTkTextbox go to end

### DISCUSSION THREAD

**Discussion by Nomanpour** [TomSchimansky/CustomTkinter#297] [General]
Hello,

Is there any method to scroll to the end of the text box after insert?

**Comment by MichelDiz** [comment]
After the insert you should do
```
self.textbox.yview_moveto(NUMBER)
```
Number is the value you wanna move. That works with `customtkinter.CTkTextbox.`

**Comment by Akascape** [comment]
```python
TextBox.see(tkinter.END)
```
